### ISSUE TITLE
[Accessibility]A11y_ Java Tooling for IntelliJ/Eclipse_ Eclipse Azure Device login_Screen Reader: When focus is on the 'Device Login' link NVDA is not reading anything

### ISSUE THREAD

**Issue by Tanushagottipati** [microsoft/azure-tools-for-java#3320]
[Check out Accessibility Insights!](https://nam06.safelinks.protection.outlook.com/?url=https%3A%2F%2Faccessibilityinsights.io%2F&data=02%7C01%7Cv-tagott%40microsoft.com%7Cb67b2c4b646d4f9561a208d6f4b5c39b%7C72f988bf86f141af91ab2d7cd011db47%7C1%7C0%7C636965458847261333&sdata=PJMiKjb64uwhBN98LZpctnBiAVeOvw%2BLBrnxBtrjl2M%3D&reserved=0) - Identify accessibility bugs before check-in and make bug fixing faster and easier

GitHubTags: #A11yTCS; #A11yMAS; #DesktopApp; #Win10; #A11y_Java Tooling for IntelliJ/Eclipse-Stay Healthy; #MAS1.3.1; #Screen Reader; #C+E-H; #Java tooling for IntelliJ/Eclipse; #NVDA

**Environment Details:**
Eclipse IDE for Enterprise Java Developers.
Version: 2019-06 (4.12.0)
Build id: 20190614-1200
Solution Installed: The Azure Toolkit for Eclipse 3.17.0

**OS Details:** 
OS: Windows (Version 1903, Build: 18362.175)
Screen Reader: NVDA
MAS Violated: MAS1.3.1

**Repro Steps:** 
1) Start NVDA and Open Java EE - Eclipse application
2) Install 'Azure Toolkit for Eclipse' from Marketplace
3) Tab till tools-->Azure-->Azure Sign in and press enter
4) Select 'Device login' and tab till sign in and press enter
5) Verify When focus is on the link, NVDA is reading the 'Device Login' link

**Actual Result:** 
When focus is on the 'Device Login' link NVDA is not reading anything

**Expected Result:** 
When focus is on the 'Device Login' link NVDA is should read the link details to the user

**User Impact:** 
Screen Reader users will not know that the focus is on the link is NVDA does not read anything

**MAS Reference:** 
https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={54f28d1f-a2d1-4dcd-84e1-5c9b87e8aba4}

![Bug_6_When focus is on the link NVDA is not reading anything](https://user-images.githubusercontent.com/41109135/62203906-b8a24d00-b3a9-11e9-86da-cc8960f58830.png)

**Comment by Flanker32** [comment]
This seems to be an issue of Eclipse Link conpoment, it could be read by NVDA with tab only once, here is the gif of Eclipse perferences ui.

![3320](https://user-images.githubusercontent.com/12445236/64110686-6643ba00-cdb5-11e9-9a4f-ade701777ef3.gif)

**Comment by Flanker32** [comment]
Importance of experience: Low, users could open the link directly by click button `Copy&Open`
Quality of experience: Low, NVDA could read the link for the first time and users could open the link directly by  click button `Copy&Open`
Evidence: All evidence is provided in this issue and external issue.

**Comment by Flanker32** [comment]
@Tanushagottipati Please have a review.

**Comment by soumyatippapuram** [comment]
@Flanker32 Please provide the link to your AAT. This is required to follow Exclusion process. For more details, please refer : https://microsoft.sharepoint.com/teams/ceacc/SitePages/External%20Bug%20Process.aspx

**Comment by Flanker32** [comment]
The issue is tracking in AAT [here](https://cela-ra-aat-prod.azurewebsites.net/#/workflow/56452990-0d38-45bc-b475-a62832ce0076), @soumyatippapuram please help us review it.

**Comment by Tanushagottipati** [comment]
External Bug: https://bugs.eclipse.org/bugs/show_bug.cgi?id=550840

**Comment by Flanker32** [comment]
Discussed this bug with Ryan and Sarah and approved to close this issue as its external and there is non blocking scenario.

**Comment by vscjavaci** [comment]
Track internal issue by AB#1749743
